src/main/resources/resources/logisim/strings/gui/gui_es.properties

```diff
@@ -332,6 +332,11 @@ editPasteItem = Pegar
 editRedoItem = Rehacer %s
 editSelectAllItem = Seleccionar todo
 editUndoItem = Deshacer %s
+editUndoHistoryMenu=Historial de Deshacer
+editRedoHistoryMenu=Historial de Rehacer
+editClearHistoryAction=Borrar Historial de Deshacer/Rehacer
+clearHistoryWarningTitle=Borrar Historial
+clearHistoryWarningMessage=¿Está seguro de que desea borrar permanentemente el historial de deshacer/rehacer?\nEsto no se puede deshacer.
```

Comment:
Use lower case for "Deshacer" and "Rehacer":

"Deshacer" -> "deshacer"
"Rehacer" -> "rehacer"

In lines 337 and 338, also use lower case for "Historial":

"Historial" -> "historial"